client/src/Components/local/molecules/json-editor/helpers/mapColumnToClassName.ts

```diff
@@ -0,0 +1,26 @@
+type UiSchema = Record<string, any>;
+
+const mapColumnToClassName = (uiSchema: UiSchema): UiSchema => {
```

Comment:
@lilitkarapetyan - could you insert a comment explaining what this does please? 

In the comment, please provide a before/after example showing what has changed.